File: Crawler Log Folder.py
class Solution(object):
    def minOperations(self, logs):
        """
        :type logs: List[str]
        :rtype: int
        """
        stack=[]
        for i in range(len(logs)):
            if logs[i] =="./":
                continue
            elif logs[i]=="../":
                if stack:
                    stack.pop()
            else:
                stack.append(logs[i])
        return len(stack)

File: Java/Crawler Log Folder.java
class Solution {
    public int minOperations(String[] logs) {
    int counter =0;
    for ( int i =0 ; i<logs.length;i++){
        if ( logs[i].equals("../")){
            if(counter!=0){
                counter--;
            }
        }
        else if(logs[i].equals("./")){
            continue;
        }
        else{
            counter++;
        }
        
    }
    return counter;
        

    }
}
